web/admin.py

```
@@ -623,6 +624,38 @@ class LearningStreakAdmin(admin.ModelAdmin):
+        (None, {
+            "fields": ("user", "course", "platform", "share_url", "verification_token", "status")
+        }),
+        ("Discount Details", {
+            "fields": ("discount_amount",)
+        }),
+        ("Dates", {
+            "fields": ("created_at", "verified_at", "expires_at", "used_at")
+        }),
+    )
+    
+    def mark_as_verified(self, request, queryset):
```

Ruff (0.8.2)

Missing return type annotation for public function `mark_as_verified`

Add return type annotation: `None`

(ANN201)